lib/workers/pr/automerge.ts

```diff
@@ -26,11 +26,11 @@ export async function checkAutoMerge(
 ): Promise<AutomergePrResult> {
   logger.trace({ config }, 'checkAutoMerge');
   const {
-    branchName,
-    automergeType,
-    automergeStrategy,
     automergeComment,
-    requiredStatusChecks,
+    automergeStrategy,
+    automergeType,
+    branchName,
+    ignoreTests,
```

Comment:
Please restore the previous ordering so that there's less diff

Reply:
Done